manifests/rhoai/addon/apps/starburst/starburst-galaxy-quickstart.yaml

```diff
@@ -59,24 +59,24 @@ spec:
 
     - title: Launch Jupyter from Red Hat OpenShift AI
       description: |-
-        ### To launch a Jupyter notebook server from within the Red Hat OpenShift AI console:
+        ### To launch a Jupyter workbench from within the Red Hat OpenShift AI console:
         1. Click **Applications** &#x2192; **Enabled**.
         2. Find the **Jupyter** card.
```

Comment:
Same question about the "Jupyter card" name. If changing above, also change here.